data-serving/data-service/src/model/location.ts

```diff
@@ -1,21 +1,29 @@
 import mongoose from 'mongoose';
 
+export enum GeoResolution {
+    Point = 'Point', // Lat/long coordinates
+    Admin3 = 'Admin3', // ~City
```

Comment:
nit: these comments feel a little antithetical to the purpose of the administrative area designation ("~" totally granted). Might be more useful to drop a link to [this article](https://en.wikipedia.org/wiki/List_of_administrative_divisions_by_country).

Reply:
wow, on this article we can see that France with its 6 admin areas (!) isn't hierarchical in ways that affect me negatively. /sigh

Reply:
removed and added link, thanks!